components/user-mgt/org.wso2.carbon.identity.user.pre.update.profile.action/src/test/resources/dbscripts/h2.sql

```diff
@@ -0,0 +1,1747 @@
+CREATE TABLE IF NOT EXISTS IDN_BASE_TABLE (
```

Comment:
Do we need to have all the tables ?

Reply:
Actually this test case doesn't interact with the data layer. However, @WithRealmService annotation seems to work only if InMemory H2 db is initialized.
That's the reason to use the script.
And as this component have any specific interactions with the data layer added the default script the product has.